bindings/wasm/tests/key_id_storage.ts

```diff
@@ -0,0 +1,113 @@
+const assert = require("assert");
+import { CoreDID, KeyIdStorage, KeyPair, KeyType, MethodDigest, VerificationMethod } from "../node";
+
+describe("Method digest", () => {
+    it("should have consistent hashing", () => {
+        let verificationMethodJson = {
+            id: "did:example:HHoh9NQC9AUsK15Jyyq53VTujxEUizKDXRXd7zbT1B5u#frag_1",
+            controller: "did:example:HHoh9NQC9AUsK15Jyyq53VTujxEUizKDXRXd7zbT1B5u",
+            type: "Ed25519VerificationKey2018",
+            publicKeyMultibase: "zHHoh9NQC9AUsK15Jyyq53VTujxEUizKDXRXd7zbT1B5u",
+        };
+
+        let verificationMethod = VerificationMethod.fromJSON(verificationMethodJson);
+        let methodDigest = new MethodDigest(verificationMethod);
+
+        let packed = methodDigest.pack();
+        let packedExpected = new Uint8Array([0, 74, 60, 10, 199, 76, 205, 180, 133]);
+        assert.deepStrictEqual(packed, packedExpected);
```

Comment:
Could you write a comment explaining that the purpose of this test is to ensure that the computed method digest for a given method is the same for Rust and Wasm and also write where the corresponding Rust test can be found?